mridata-3dfse-knee/unet/supervised/Supervised_Aug_UNET_14_Scan.yaml

```diff
@@ -0,0 +1,52 @@
+# Inherit fields from the unet.yaml config.
+_BASE_: "../../templates/unet.yaml"
+# DATALOADER has 14 supervised scans
+DATALOADER:
+  SUBSAMPLE_TRAIN:
+    NUM_TOTAL: 14
+    NUM_UNDERSAMPLED: 0
+# Specify an output directory that is unique for this experiment.
+OUTPUT_DIR: results://vortex/mridata_knee_3dfse/unet/supervised/supervised_aug_unet_14_scan
+DESCRIPTION:
+  # Use f-strings to specify reading values in braces ({}) the config file.
+  # This can only be done for string fields.
+  BRIEF: f"Supervised Aug U-Net 14 Scan {AUG_TRAIN.UNDERSAMPLE.ACCELERATIONS}x mridata - {MODEL.SSDU.MASKER.PARAMS.kind}/loss={MODEL.RECON_LOSS.NAME}" 
+  EXP_NAME: vortex/mridata_knee_3dfse/unet/supervised_aug_unet_14
+  TAGS:
+  - baseline
+  - supervised
+  - 14_scan
+  - 16x
+  - mridata_knee_3dfse
+  - unet
+# For no motion aug, A2R and Consistency was removed.
+# For UNET, kept META_ARCHITECTURE as UnetModel
+MODEL:
+  A2R:
+    META_ARCHITECTURE: UnetModel 
+    USE_SUPERVISED_CONSISTENCY: false  # TUNE 
```

Comment:
Remove this - not used for supervised training

Reply:
Fixed, thanks!